index.html

```diff
@@ -0,0 +1,86 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>Kyam Harris | Portfolio</title>
+	<link rel="stylesheet" type="text/css" href="normalize.css">
+	<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+	
+<body>
+	<header>
+		<nav class="container">
+			<ul class="navLinks">
+				<li><a href="#about">About</a></li>
+				<li><a href="portfolio.html" id="portFolio">Portfolio</a></li>
+				<li><a href="#foot">Contact</a></li>
+			</ul>
+			<div class="title">
+				<h1>Kyam Harris</h1>
+				<h4>Placeholder text, maybe skills and experience</h4>
+
+			</div>
+			<section class="social">
```

Comment:
Maybe make it another `ul` like your `.navlinks`

Reply:
Completed